debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/SourceInfo.java

```diff
@@ -199,15 +231,16 @@ protected Schema schema() {
      * Get a {@link Struct} representation of the source {@link #partition()} and {@link #offset()} information. The Struct
      * complies with the {@link #SCHEMA} for the Postgres connector.
      * <p>
-     * This method should always be called after {@link #update(Long, Long, Integer)}.
+     * This method should always be called after {@link #update(Long, Long, Long, TableId)}.
      *
      * @return the source partition and offset {@link Struct}; never null
      * @see #schema()
      */
     protected Struct source() {
```

Comment:
Taking the remark about the offset from above into account, the new fields must be explicitly added here.